### PR TITLE
Fix issue606

### PR DESCRIPTION
**Pull request by peiying16**
Fix issue 606 on bracket side
https://github.com/mozilla/brackets/issues/606

### CODE REVIEW

**Comment by humphd** [comment]
Nice fix, I love that we're moving to using the pref system over local variables.  Smart.

**Comment by Pomax** [comment]
Agreed - do we still need the `jsEnabledOverride` flag with this change, or can we simply make sure that by default this pref is `true`? (it'd be nice to, at this point in time, remove that option again if we can)

**Comment by humphd** [comment]
The override stuff is a separate code path that we still need, yes.

File: src/filesystem/impls/filer/lib/HTMLRewriter.js [hunk 1]
```diff
@@ -140,7 +135,7 @@ define(function (require, exports, module) {
                 return;
             }
 
-            if(jsEnabled || jsEnabledOverride) {
+            if(PreferencesManager.get("allowJavaScript") || jsEnabledOverride) {
```
Comment (gideonthomas) [inline src/filesystem/impls/filer/lib/HTMLRewriter.js]: small optimization here:
store the result of `PreferencesManager.get("allowJavaScript")` in a variable (maybe `allowJavascript`) on line 131 and use that variable here instead.

**Comment by peiying16** [comment]
Did it, I use the previous variable  jsEnabled.

**Comment by gideonthomas** [comment]
Thanks for the contribution @peiying16! Much appreciated!

**Pull request merged**